src/__tests__/index.js

```diff
@@ -89,6 +89,7 @@ pluginTester({
         // @codegen
         /* comment */`,
     },
+    'codegen`module.exports = "var ALLCAPS = \'ALLCAPS\'"`',
```

Comment:
I added this test for the ALLCAPS template regression in babel7.
I guess the plugin test is using babel 6? Can we somehow test this?

Reply:
We would need to refactor https://github.com/babel-utils/babel-plugin-tester to be able to accept babel-core as an option so we can provide our own version.